lib/compat/wordpress-6.0/get-global-styles-and-settings.php

```diff
@@ -85,7 +86,7 @@ function gutenberg_get_global_styles_svg_filters() {
 		}
 	}
 
-	$supports_theme_json = wp_theme_has_theme_json();
+	$supports_theme_json = wp_theme_has_theme_json( $stylesheet );
```

Comment:
Do we need to do this?

Reply:
This saves a another lookup to `get_option`. We have the stylesheet to hand, so why not just pass it in and save a lookup to cache.